examples/validate_relative_binding.py

```diff
@@ -247,215 +244,76 @@ def fn(*args, **kwargs):
         cmd_args.num_complex_preequil_steps,
     )
 
-    complex_conversion_schedule = construct_conversion_lambda_schedule(cmd_args.num_complex_conv_windows)
-    # complex models
-    binding_model_complex_conversion = model_rabfe.AbsoluteConversionModel(
-        client,
-        forcefield,
-        complex_system,
-        complex_conversion_schedule,
-        complex_topology,
-        temperature,
-        pressure,
-        dt,
-        cmd_args.num_complex_equil_steps,
-        cmd_args.num_complex_prod_steps,
-        frame_filter=frame_filter,
-    )
 
-    binding_model_complex_decouple = model_rabfe.RelativeBindingModel(
+    rabfe_model = model_rabfe.RABFEModel(
```

Comment:
This RABFEModel is constructed using all of the static variables of the model, most notably: the blocker, the complex config and the solvent config.